raiden/blockchain/abi.py

```diff
@@ -70,6 +78,18 @@ def get_eventname_types(event_description):
     return name, encode_types
 
 
+#  Initially, I wanted to use solc's AST output, but that thing is so crazy that parsing
```

Comment:
On second thought... just remove the entire comment. Serves no purpose. The decision to go with a regex is fine, so no need for the comment.